pkg/descriptions.json

```diff
@@ -1204,6 +1204,14 @@
         ]
     },
     {
+        "Name": "System.Runtime.Serialization.Formatters",
+        "Description": "Provides common types for libraries that support binary serialization.",
```

Comment:
Not just binary serialization, right?  Any IFormatter that uses these core formatter/serialization-related types?